hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/CacheAwareLoadBalancer.java

```diff
@@ -53,6 +55,12 @@
 public class CacheAwareLoadBalancer extends StochasticLoadBalancer {
   private static final Logger LOG = LoggerFactory.getLogger(CacheAwareLoadBalancer.class);
 
+  public static final String CACHE_RATIO_THRESHOLD = "hbase.master.balancer.stochastic.cacheRatio";
+
+  public static final String MOVE_THROTTLING = "hbase.master.balancer.move.throttlingMs";
+
+  public static final long MOVE_THROTTLING_DEFAULT = 60 * 1000;
```

Comment:
nit: the unit is ambiguous. We should either use a more intentional type like Duration, or we should include the unit in the name

Reply:
Decided to go with using the Duration suggestion. This is now moved to the LoadBalancer interface (see my reply in other comment above).